R/plotting/access_change_income_areas.R

```diff
@@ -40,7 +40,7 @@ agents <- agents %>%
 
 # Plot ----
 
-income_name <- c("low 10 %", rep("", 8), "high 10 %")
+income_names <- c("min 10 %", rep("", 8), "max 10 %")
```

Comment:
If I remember the context right, should this be:
"min 10 %" -> "alin 10 %"
"max 10 %" -> "ylin 10 %"